client/src/styles/_helpers.scss

```diff
@@ -1,3 +1,10 @@
+// Mobile related styles
+@mixin mobile {
```

Comment:
Nice.

How did we not thing about doing this instead of ` @mixin breakpoint(size)` ??

Reply:
I don't know but it's indeed easier this way.